deploy/manifests/dev/us-east-2/tenant/storetheindex/kustomization.yaml

```diff
@@ -10,13 +10,7 @@ resources:
 patchesStrategicMerge:
   - patch-indexer.yaml
 
-configMapGenerator:
-  - name: indexer-env-vars
-    behavior: merge
-    literals:
-      - STORETHEINDEX_PRIV_KEY_PATH=/identity/identity.key
-
 secretGenerator:
   - name: indexer-identity
     files:
-      - identity.key=identity.encrypted
+      - indexer-0.key=indexer-0-identity.encrypted
```

Comment:
will we need one of these for each instance?

Reply:
yes